src/assets/styles/components/task-details.scss

```diff
@@ -1,5 +1,69 @@
-.task-details-container {
-  margin-block: 48px;
-  background-color: var(--surface);
-  width: 765px;
+.task-details {
+  padding: 20px;
+  background-color: var(--ds-background-neutral);
+  height: 100%;
+
+  .task-header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 36px;
+
+    .title {
+      display: flex;
+      align-items: center;
+      gap: 10px;
+      width: 100%;
+    }
+  
+    .title-input {
+      display: inline-flex;
+      font-size: 20px;
+      background: transparent;
+      width: 88%;
+      color: var(--ds-text);
+      padding: 2px 2px;
+      outline: none;
+      border:none;
+      border-radius: 5px;
+  
+      &:focus {
+        border: 2px solid var(--ds-border-focused);
+        background-color: var(--surface);
+      }
+    }
+
+    p {
+      padding:0;
+      margin:5px 0 0 75px;
+      width: 100%;
+    }
+  }
+
+  .task-body {
+    display: grid;
+    grid-template-columns: 3fr 1fr;
```

Comment:
the grid dont looks, right. also please check the responsive in trello.

Reply:
@JonathanYesh279 I think you need to add responsive, look the dialog in trello. under 750px
<img width="742" alt="image" src="https://github.com/user-attachments/assets/6dcfbd53-dccf-4514-a3ab-8acd111b8217">

Reply:
I've added a media queris to this container and made it responsive under 750px, make it two columns at the bottom of the page.

Reply:
move the media query inside the class. look my comment above